companies/proofpoint.json

```diff
@@ -0,0 +1,38 @@
+{
+    "slug": "proofpoint",
+    "relevant-countries": [
+        "all"
+    ],
+    "categories": [
+        "telecommunication"
+    ],
+    "name": "Proofpoint, Inc",
+    "runs": [
```

Comment:
The privacy policy lists a couple of affiliated companies/subsidiaries that it also applies to:

Cloudmark, LLC, ObserveIT, Ltd., Proofpoint Canada, Inc., Proofpoint Limited (UK), Proofpoint Pty Ltd (Australia), Proofpoint Japan KK, Proofpoint GmbH (Germany), Proofpoint Singapore Ptd. Ltd., Proofpoint Int’l (Taiwan), and Wombat Security Technologies, Inc.